lib/bolt/cli.rb

```diff
@@ -725,19 +708,17 @@ def generate_types
     #
     def install_project_modules(project, config, force, resolve)
       assert_project_file(project)
-      assert_puppetfile_or_module_command(project.modules)
 
-      unless project.modules
+      unless project.modules.any?
```

Comment:
This will error if `modules` is not configured. If we want `modules` to have a default `[]`, then `Bolt::Project.modules` needs to be updated.

Reply:
Ok, updated to set the default in the project as well.I think we'll end up removing most references to `project.modules` once we don't need to modify the modulepath, but we'll need it for the migrator.